src/app/public/testing/radio-fixture.ts

```diff
@@ -0,0 +1,188 @@
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Allows interaction with a SKY UX radio buttons within a radio group.
+ */
+export class SkyRadioFixture {
+
+  private debugEl: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(this.fixture, this.skyTestId, 'sky-radio-group');
+  }
+
+  /**
+   * The selected radio button value.
+   */
+  public get value(): any {
+    const selectedRadio = this.debugEl.query(By.css('sky-radio input:checked'));
+    const selectedValue = selectedRadio && selectedRadio.nativeElement.value;
+
+    return selectedValue;
+  }
+
+  /**
+   * The selected radio button label.
+   */
+  public get selectedLabel(): string {
+    const allRadioButtons = this.getAllSkyRadioButtonEls();
+    const selectedRadioInputValue = this.value;
+
+    for (let i = 0; i < allRadioButtons.length; i++) {
+      const skyRadio = allRadioButtons[i];
+      const inputEl = skyRadio.query(By.css('input'));
+
+      if (inputEl.nativeElement && inputEl.nativeElement.value === selectedRadioInputValue) {
+        return this.getRadioButtonLabelText(i);
+      }
+    }
+  }
+
+  /**
+   * Get the value of the radio button at the given index.
+   * @param index The index of the radio button.
+   */
+  public getRadioButtonValue(index: number): any {
+    const radioButton = this.getRadioButtonInputEl(index);
+
+    return radioButton && radioButton.nativeElement.value;
+  }
+
+  /**
+   * A flag indicating if every radio button in the radio group is disabled.
+   */
+  public get allRadioButtonsDisabled(): boolean {
```

Comment:
For MVP, we should assume the consumer is using the radio group "properly", by wrapping all radio buttons within a `sky-radio-group`. That being the case, we could simplify all of the "disabled" properties to: `get disabled` and `set disabled` (since all radio buttons would be disabled if the parent radio group is disabled).

(We could even throw an error if a `sky-radio-group` isn't found.)